### PR TITLE
[13.0][IMP] fieldservice: request_late (Forward port from 12.0)

### PR DESCRIPTION
**Pull request by HekkiMelody**
Forward port of #815 #817

### CODE REVIEW

File: fieldservice/models/res_company.py [hunk 1]
```diff
@@ -14,3 +14,20 @@ class ResCompany(models.Model):
         string="Auto-populate Equipments on Order based on Location"
     )
     search_on_complete_name = fields.Boolean(string="Search Location By Hierarchy")
+
+    order_prio0_request_late = fields.Float(
```
Comment (max3903) [inline fieldservice/models/res_company.py]:
```suggestion
    fsm_order_request_late_lowest = fields.Float(
```

File: fieldservice/models/res_company.py [hunk 2]
```diff
@@ -14,3 +14,20 @@ class ResCompany(models.Model):
...
+        string="Hours of Buffer for Lowest Priority FS Orders",
+        default=72,
+    )
+    order_prio1_request_late = fields.Float(
```
Comment (max3903) [inline fieldservice/models/res_company.py]:
```suggestion
    fsm_order_request_late_low = fields.Float(
```

File: fieldservice/models/res_company.py [hunk 3]
```diff
@@ -14,3 +14,20 @@ class ResCompany(models.Model):
...
+        string="Hours of Buffer for Low Priority FS Orders",
+        default=48,
+    )
+    order_prio2_request_late = fields.Float(
```
Comment (max3903) [inline fieldservice/models/res_company.py]:
```suggestion
    fsm_order_request_late_medium = fields.Float(
```

File: fieldservice/models/res_company.py [hunk 4]
```diff
@@ -14,3 +14,20 @@ class ResCompany(models.Model):
...
+        string="Hours of Buffer for Medium Priority FS Orders",
+        default=24,
+    )
+    order_prio3_request_late = fields.Float(
```
Comment (max3903) [inline fieldservice/models/res_company.py]:
```suggestion
    fsm_order_request_late_high = fields.Float(
```

**Review by max3903 — APPROVED** [review]

**Comment by HekkiMelody** [comment]
I'll also provide a quick migration script

**Pull request closed without merging**